modules/platforms/python/pyignite/api/key_value.py

```diff
@@ -993,3 +1005,64 @@ def cache_get_size(
     if result.status == 0:
         result.value = result.value['count']
     return result
+
+
+def cache_local_peek(
```

Comment:
Is it a public API shown to a user?

Reply:
No, it is private. The `api` module contain functions that represent corresponding Ignite thin protocol operations.